js/src/dom/manipulator.js

```diff
@@ -47,9 +47,13 @@ const Manipulator = {
       ...element.dataset
     }
 
-    Object.keys(attributes).forEach(key => {
-      attributes[key] = normalizeData(attributes[key])
-    })
+    Object.keys(attributes)
+      .filter(attributes => attributes.startsWith('bs'))
```

Comment:
BTW this should be `attribute` probably?

Reply:
Yes :smile: updated